includes/admin/post-types/meta-boxes/class.llms.meta.box.voucher.php

```diff
@@ -225,10 +227,10 @@ private function redemption_section_html() {
 						$user = get_user_by( 'id', $redeemed_code->user_id );
 						?>
 						<tr>
-							<td><?php echo $user->data->display_name; ?></td>
-							<td><?php echo $user->data->user_email; ?></td>
-							<td><?php echo $redeemed_code->redemption_date; ?></td>
-							<td><?php echo $redeemed_code->code; ?></td>
+							<td><?php echo esc_attr( $user->data->display_name ); ?></td>
+							<td><?php echo esc_attr( $user->data->user_email ); ?></td>
+							<td><?php echo esc_attr( $redeemed_code->redemption_date ); ?></td>
+							<td><?php echo esc_attr( $redeemed_code->code ); ?></td>
```

Comment:
I'd go with `esc_html()` here, what do you think?

Reply:
oops! corrected. Haste makes waste 😅